hvpy/api_groups/jpeg2000/tests/test_get_jpx.py

```diff
@@ -0,0 +1,59 @@
+from datetime import datetime
+
+import pytest
+from pydantic import ValidationError
+
+from hvpy.api_groups.jpeg2000.get_jpx import getJPXInputParameters
+from hvpy.core import execute_api_call
+
+startTime = datetime(2014, 1, 1, 0, 0, 0)
+endTime = datetime(2014, 1, 1, 0, 45, 0)
```

Comment:
Can you remove these and just do `startTime=datetime(2014, 1, 1, 0, 0, 0)` etc?

Reply:
do those where?

Reply:
Just define them in the test.